### PR TITLE
fix(languages,i18n): fix i18n,FormattedMessage

### PR DESCRIPTION
**Pull request by gautism1**
self closing tags and ids in language folder

Made a new branch ,resolving the bugs mentioned

### CODE REVIEW

File: src/pages/Customers/AddCustomer.js [hunk 1]
```diff
@@ -1,56 +1,61 @@
...
         } catch (e) {
             console.error(e);
         }
-    }, [fields]);
+    };
 
     return (
         <Fragment>
-            <Navbar opened="customers" />
+            <Navbar opened="inventory" />
```
Comment (rparsh) [inline src/pages/Customers/AddCustomer.js:44]: why changed this?

**Review by rparsh — APPROVED** [review]

File: src/pages/Customers/AddCustomer.js (hunk 1 above)
Comment (rparsh) [reply src/pages/Customers/AddCustomer.js:44]: also new Customer opening new Item Page @gautism1

**Review by rparsh — CHANGES_REQUESTED** [review]
new customer button opening new item component

**Review by rparsh — APPROVED** [review]

**Comment by gautism1** [comment]
ok

**Pull request merged**